test/index.js

```diff
@@ -549,6 +548,157 @@ describe("mongoose_delete with options: { deletedBy : true, deletedByType: Strin
     });
 });
 
+describe("mongoose_delete with options: { deletedId : true }", function () {
+
+    var deletedIdSchema1 = new Schema({name: String}, {collection: 'mongoose_delete_test_deleted_id_1'});
+    deletedIdSchema1.plugin(mongoose_delete, { deletedId: true });
+    var TestDeletedId1 = mongoose.model('TestDeletedId1', deletedIdSchema1);
+
+    var puffy1 = null;
+    var puffy2 = null;
+
+    beforeEach(async function () {
+        const created = await TestDeletedId1.create(
+          [
+              { name: 'Puffy1' },
+              { name: 'Puffy2' },
+              { name: 'Puffy3', deleted: true, deletedId: "53da93b16b4a6670076b16bf" }
+          ]
+        );
+
+        puffy1 = { ...created[0]._doc };
+        puffy2 = { ...created[1]._doc };
+    });
+
+    afterEach(async function () {
+        await mongoose.connection.db.dropCollection("mongoose_delete_test_deleted_id_1");
+    });
+
+    var userId = getNewObjectId("53da93b16b4a6670076b16bf");
+
+    it("delete() -> should save 'deletedId' key", async function () {
+        try {
+            const puffy = await TestDeletedId1.findOne({ name: 'Puffy1' });
+            const success = await puffy.delete( {deletedId: userId});
```

Comment:
🤏 I wouldn't put `userId` here as that will lead people to ask why even bother to have a `deletedId`. I'd call it `actionId` if you want to give a hint as to why it's needed or just call it `deletedId` when you declare it.